x/auth/middleware/sigverify.go

```diff
@@ -299,6 +299,10 @@ type sigGasConsumeTxHandler struct {
 // CONTRACT: Pubkeys are set in context for all signers before this middleware runs
 // CONTRACT: Tx must implement SigVerifiableTx interface
 func SigGasConsumeMiddleware(ak AccountKeeper, sigGasConsumer SignatureVerificationGasConsumer) tx.Middleware {
+	if sigGasConsumer == nil {
+		sigGasConsumer = DefaultSigVerificationGasConsumer
```

Comment:
Would it be possible to now make `DefaultSigVerificationGasConsumer` private as well?

Reply:
I guess so, technically. But I wouldn't do it for 2 reasons:
1. we're in beta, would like to minimize API breaking changes
2. `DefaultSigVerificationGasConsumer` can be extended. E.g. a chain can create a wrapper over that function that's consumes gas for a certain new pubkey type they support, but then call this SDK one for all remaining pubkey types.

Reply:
@crodriguezvega what is the reasoning to make it private?

Reply:
I was just wondering if it was possible, so that the code looks the same as for `DeductFeeMiddleware` and `NewExtensionOptionsMiddleware`. But I am completely fine with the way it is. 

Thanks for the explanation, @amaurym!